install-with-docker/003-second-node.md

```diff
@@ -1,8 +1,33 @@
-Now switch to the second node and install the container again:
+Now switch to the second node and install the init container again:
 
-`docker run -d --name storageos -e HOSTNAME=host02 -e ADVERTISE_IP=[[HOST2_IP]] -e JOIN=[[HOST_IP]] --net=host --pid=host --privileged --cap-add SYS_ADMIN --device /dev/fuse -v /var/lib/storageos:/var/lib/storageos:rshared -v /run/docker/plugins:/run/docker/plugins storageos/node:0.10.0 server `{{execute T2}}
+`docker run --name enable_lio                 \
+           --privileged                       \
+           --rm                               \
+           --cap-add=SYS_ADMIN                \
+           --volume /lib/modules:/lib/modules \
+           --volume /sys:/sys:rshared         \
+           storageos/init:0.1`{{execute T2}}
 
-Confirm that the installation was successful:
+Now you can install the StorageOS node container on the host.
+
+In the second host terminal:
```

Comment:
In my browser, the \ characters don't line up - I think they are being rendered using a proportional space font. Might be worth cleaning the spaces up so there is a single ' \' after each stanza, rather than attempting to line things up?